rules/travis.js

```diff
@@ -74,15 +75,23 @@ module.exports = ctx => {
 				});
 			}
 
-			const versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+			let versions = Array.isArray(travis.node_js) ? travis.node_js : [travis.node_js];
+
+			versions = versions.filter(v => IGNORED_VERSIONS.indexOf(v) === -1);
 
 			for (const version of versions) {
 				if (DEPRECATED_VERSIONS.indexOf(version) !== -1) {
 					ctx.report({
 						message: `Version \`${version}\` is deprecated.`,
 						file
 					});
-				} else if (version !== 'node' && engine && !semver.satisfies(normalize(version), engine)) {
+				}
+			}
+
+			versions = versions.filter(v => DEPRECATED_VERSIONS.indexOf(v) === -1);
```

Comment:
I could have merged this filter with the above for…of loop, however, I think that the code is more readable this way and the performance impact is negligible.